src/files/FootnotesXml.ts

```diff
@@ -160,6 +160,14 @@ export class FootnotesXml extends XmlFileWithContentTypes {
 								)
 							);
 						} else {
+							if (
+								typeof firstNode.children[0].children[0] ===
+								'string'
+							) {
+								firstNode.children[0].children[0] = ' '.concat(
+									firstNode.children[0].children[0]
+								);
+							}
```

Comment:
`firstNode.children[0].children[0]` happens to be `image`. You can use that node to make the update. And perhaps it also makes sense to change the variable name 😬.

```javascript
	if (firstNode instanceof Paragraph) {
		// Check if the first child node is an image.
		const [text] = firstNode.children;
		const [imageOrText] = text
			? text.children
			: [undefined];
		if (imageOrText && imageOrText instanceof Image) {
			// ommited for clarity
		} else {
			if (
				imageOrText &&
				typeof imageOrText === 'string'
			) {
				text.children[0] = ' '.concat(imageOrText);
			}

			firstNode.children.unshift(
				new FootnoteAnchor({
					style: footnote.style,
				})
			);
		}
	}
```